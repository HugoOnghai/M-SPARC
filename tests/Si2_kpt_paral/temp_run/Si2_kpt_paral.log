
                            < M A T L A B (R) >
                  Copyright 1984-2019 The MathWorks, Inc.
                  R2019a (9.6.0.1072779) 64-bit (glnxa64)
                               March 8, 2019

 
To get started, type doc.
For product information, visit www.mathworks.com.
 

 Reading .inpt file ...


<INPT>
# $ cat Si2_kpt_paral.inpt
# nprocs: 4
FD_ORDER: 12
BOUNDARY_CONDITION: 2
EXCHANGE_CORRELATION: GGA_PBE

TOL_SCF: 1e-6
TOL_POISSON: 1e-8
TOL_PSEUDOCHARGE: 1e-8

ELEC_TEMP: 315.775131
ELEC_TEMP_TYPE: fd


CALC_STRESS: 1

RELAX_MAXDILAT: 1.2
PRINT_ATOMS: 1
PRINT_FORCES: 1

CELL: 7.5 7.5 7.5   ## test comment
FD_GRID: 25 25 25         # FD grid

LATVEC:
0.5 0.5 0.0
0.0 0.5 0.5
0.5 0.0 0.5

KPOINT_GRID: 2 2 2

NP_KPOINT_PARAL: 4
<\INPT>

Warning: "BOUNDARY_CONDITION" is obsolete, use "BC" instead!
Neglecting option "NP_KPOINT_PARAL:", which is not supported in M-SPARC
 Reading .ion file ...
 Number of atom types : 1
 Total number of atoms: 2
 Default atomic mass for Si is 28.085000
 pseudo_dir # 1 = Si.psp8

 COORD:
                   0                   0                   0
   1.875000000000000   1.875000000000000   1.875000000000000

   0.030168927102176

 kpoint grid before symmetry:
   0.209439510239320   0.209439510239320   0.209439510239320
  -0.209439510239320   0.209439510239320   0.209439510239320
   0.209439510239320  -0.209439510239320   0.209439510239320
  -0.209439510239320  -0.209439510239320   0.209439510239320
   0.209439510239320   0.209439510239320  -0.209439510239320
  -0.209439510239320   0.209439510239320  -0.209439510239320
   0.209439510239320  -0.209439510239320  -0.209439510239320
  -0.209439510239320  -0.209439510239320  -0.209439510239320

 kpoint grid after symmetry:
   0.209439510239320   0.209439510239320  -0.209439510239320
  -0.209439510239320   0.209439510239320  -0.209439510239320
   0.209439510239320  -0.209439510239320  -0.209439510239320
  -0.209439510239320  -0.209439510239320  -0.209439510239320

## Chebyshev polynomial degree not provided, finding npl ...
## Based on the mesh size, npl is set to: 25
## Number of states not provided, finding Nev ...
## Based on the number of electrons, Nev is set to: 10
 Creating differentiation matrices ...
 Done. (3.559 sec)

 Estimated memory usage:
 Total:   81.78 MB
 orbitals            :   38.15 MB
 sparse matrices     :   40.29 MB
 global-size vectors :    1.67 MB
 mixing histories    :    1.67 MB

 Finding rb for Si ...
 rb = {3.900 3.900 3.900}, int_b = -3.999988086170051, err_rb = 1.191e-05
 rb = {4.200 4.200 4.200}, int_b = -4.000037071383748, err_rb = 3.707e-05
 rb = {4.500 4.500 4.500}, int_b = -4.000024472105587, err_rb = 2.447e-05
 rb = {4.800 4.800 4.800}, int_b = -4.000018885025834, err_rb = 1.889e-05
 rb = {5.100 5.100 5.100}, int_b = -4.000015895832916, err_rb = 1.590e-05
 rb = {5.400 5.400 5.400}, int_b = -4.000012929256292, err_rb = 1.293e-05
 rb = {5.700 5.700 5.700}, int_b = -4.000012745432590, err_rb = 1.275e-05
 rb = {6.000 6.000 6.000}, int_b = -4.000012562347726, err_rb = 1.256e-05
 rb = {6.300 6.300 6.300}, int_b = -4.000011711719052, err_rb = 1.171e-05
 rb = {6.600 6.600 6.600}, int_b = -4.000011371065260, err_rb = 1.137e-05
 rb = {6.900 6.900 6.900}, int_b = -4.000010123447447, err_rb = 1.012e-05
 rb = {7.200 7.200 7.200}, int_b = -4.000009998809864, err_rb = 9.999e-06
 rb = {7.500 7.500 7.500}, int_b = -4.000004045605484, err_rb = 4.046e-06
 rb = {7.800 7.800 7.800}, int_b = -3.999999520713764, err_rb = 4.793e-07
 rb = {8.100 8.100 8.100}, int_b = -4.000000101253654, err_rb = 1.013e-07
 rb = {8.400 8.400 8.400}, int_b = -3.999999978619319, err_rb = 2.138e-08
 rb = {8.700 8.700 8.700}, int_b = -3.999999999757385, err_rb = 2.426e-10
 rb = {8.700 8.700 8.700}

 ###############################################################
 Relaxation step number: 1 

 Starting pseudocharge generation and self energy calculation...
 Integration b = 8.000000003758

 Integration b_ref = 8.000000000017

 ****************************************
 *          Eself_ref = 57.606940       *
 ****************************************
 Done. (1.944602 s)
 Time for b calculation: 1.959 seconds.

 Starting calculating nonlocal projectors ...
 Done. (1.220339 s)

 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
 Starting SCF iteration...
 AAR converged to a relative residual of 9.89682e-09 in 34 iterations.
 Poisson problem took 1.486879s
 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 1 
 ============================================= 
 Fermi energy = 3.389376
 This SCF iteration took 20.495 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 2 
 ============================================= 
 Fermi energy = 0.461582
 This SCF iteration took 10.069 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 3 
 ============================================= 
 Fermi energy = 0.223556
 This SCF iteration took 9.741 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 4 
 ============================================= 
 Fermi energy = 0.209932
 AAR converged to a relative residual of 9.29243e-09 in 27 iterations.
 Poisson problem took 0.759391s
 Escc = 0.16792073
 Etot = -7.93070633
 Eatom = -3.96535317
 Error in SCF iteration: 2.6602e-02 
 This SCF iteration took 10.577 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  2 
 ========================= 
 Fermi energy = 0.218893
 AAR converged to a relative residual of 1.51092e-09 in 25 iterations.
 Poisson problem took 0.697908s
 Escc = 0.04098403
 Etot = -7.93704641
 Eatom = -3.96852320
 Error in SCF iteration: 1.4104e-02 
 This SCF iteration took 10.416 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  3 
 ========================= 
 Fermi energy = 0.219250
 AAR converged to a relative residual of 1.61194e-09 in 25 iterations.
 Poisson problem took 0.706117s
 Escc = -0.00882721
 Etot = -7.93791983
 Eatom = -3.96895992
 Error in SCF iteration: 3.4182e-03 
 This SCF iteration took 10.451 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  4 
 ========================= 
 Fermi energy = 0.218130
 AAR converged to a relative residual of 4.74978e-09 in 19 iterations.
 Poisson problem took 0.546623s
 Escc = -0.00347942
 Etot = -7.93794001
 Eatom = -3.96897001
 Error in SCF iteration: 1.7808e-03 
 This SCF iteration took 10.359 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  5 
 ========================= 
 Fermi energy = 0.217958
 AAR converged to a relative residual of 3.43281e-09 in 19 iterations.
 Poisson problem took 0.543357s
 Escc = -0.00002417
 Etot = -7.93794345
 Eatom = -3.96897173
 Error in SCF iteration: 3.2374e-04 
 This SCF iteration took 10.278 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  6 
 ========================= 
 Fermi energy = 0.217985
 AAR converged to a relative residual of 6.79431e-09 in 14 iterations.
 Poisson problem took 0.399367s
 Escc = -0.00009920
 Etot = -7.93794364
 Eatom = -3.96897182
 Error in SCF iteration: 1.6855e-04 
 This SCF iteration took 10.116 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  7 
 ========================= 
 Fermi energy = 0.217962
 AAR converged to a relative residual of 8.37386e-09 in 14 iterations.
 Poisson problem took 0.399061s
 Escc = -0.00000378
 Etot = -7.93794374
 Eatom = -3.96897187
 Error in SCF iteration: 5.0581e-05 
 This SCF iteration took 10.812 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  8 
 ========================= 
 Fermi energy = 0.217967
 AAR converged to a relative residual of 2.72051e-09 in 13 iterations.
 Poisson problem took 0.370037s
 Escc = 0.00000490
 Etot = -7.93794372
 Eatom = -3.96897186
 Error in SCF iteration: 1.4326e-05 
 This SCF iteration took 10.055 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  9 
 ========================= 
 Fermi energy = 0.217968
 AAR converged to a relative residual of 7.79457e-10 in 13 iterations.
 Poisson problem took 0.367081s
 Escc = 0.00000800
 Etot = -7.93794372
 Eatom = -3.96897186
 Error in SCF iteration: 3.4264e-06 
 This SCF iteration took 9.887 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 10 
 ========================= 
 Fermi energy = 0.217969
 AAR converged to a relative residual of 3.22034e-10 in 13 iterations.
 Poisson problem took 0.370144s
 Escc = -0.00000192
 Etot = -7.93794372
 Eatom = -3.96897186
 Error in SCF iteration: 1.4212e-06 
 This SCF iteration took 9.918 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 11 
 ========================= 
 Fermi energy = 0.217968
 AAR converged to a relative residual of 5.18454e-09 in 7 iterations.
 Poisson problem took 0.197816s
 Escc = 0.00000051
 Etot = -7.93794371
 Eatom = -3.96897185
 Error in SCF iteration: 2.2322e-07 
 This SCF iteration took 9.739 s.


 Finished SCF iteration in 11 steps!
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 

 **********************************************************
 *          Energy per unit cell = -7.937943708 Ha.       *
 *          Energy per atom = -3.968971854 Ha.            *
 **********************************************************

 Starting atomic force calculation ... 
 local force calculation: 2.037 s
 ***********************************************************
 *                      Atomic Force                       *
 ***********************************************************
 Drift free forces (Ha/Bohr):
   0.002442845857756   0.002442820308255   0.002442788996878
  -0.002442845857756  -0.002442820308255  -0.002442788996878

 Max magnitude of forces (Ha/Bohr):   0.004231085561230


 Time for calculating forces: 2.568484 s.

["Stress in GPa"


]   7.354082384311034   5.441965416478056   5.441970500661295
   5.441965416478056   7.354075771715887   5.441985280446382
   5.441970500661295   5.441985280446382   7.354074756042928


 Time for calculating stress: 7.679285 s.

 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
 Final atomic positions (Cartesian) are as follows:
  0.000000 	  0.000000 	  0.000000 
  1.875000 	  1.875000 	  1.875000 

 Run-time of the program: 173.026090 seconds

ans = 

  struct with fields:

                  cell_typ: 2
                   lat_vec: [3x3 double]
                  metric_T: [3x3 double]
                    grad_T: [3x3 double]
                    lapc_T: [3x3 double]
                      Jacb: 0.707106781186547
                        L1: 7.500000000000000
                        L2: 7.500000000000000
                        L3: 7.500000000000000
                        Nx: 25
                        Ny: 25
                        Nz: 25
                         N: 15625
                      ecut: -1
              mesh_spacing: -1
                   kptgrid: [4x3 double]
                  kptshift: [1x3 double]
                      nkpt: [2 2 2]
                     tnkpt: 4
                      wkpt: [4x1 double]
                        BC: 2
                       BCx: 0
                       BCy: 0
                       BCz: 0
                      isBS: 0
                   lattice: 'undefined'
                   SCF_tol: 1.000000000000000e-06
                       Nev: 10
               poisson_tol: 1.000000000000000e-08
          pseudocharge_tol: 1.000000000000000e-08
                       Cst: 27.211386245987999
              Ha_Bohr3_GPa: 2.942101569650548e+04
                        kB: 3.166811563455600e-06
               elec_T_type: 0
                      Temp: 3.157751310000000e+02
                       bet: 9.999996636976061e+02
                       npl: 25
                       FDn: 6
                    rc_ref: 0.500000000000000
              max_relax_it: 100
                dbg_switch: 0
                        xc: 2
                 Nelectron: 8
                     n_typ: 1
                TimeRevSym: 1
                  spin_typ: 0
                     nspin: 1
              CheFSI_Optmz: 0
          chefsibound_flag: 0
               FixRandSeed: 0
                rhoTrigger: 4
                 NetCharge: 0
                 MAXIT_SCF: 100
                 MINIT_SCF: 3
             MAXIT_POISSON: 1000
            accuracy_level: -1
     target_force_accuracy: -1
    target_energy_accuracy: -1
                 TOL_RELAX: 5.000000000000000e-04
               TOL_LANCZOS: 0.010000000000000
               precond_tol: 9.000000000000003e-05
               precond_kTF: 1
            precond_thresh: 0.250000000000000
            precond_fitpow: 2
          precond_resta_q0: 1.360000000000000
          precond_resta_Rs: 2.760000000000000
            MixingVariable: 1
             MixingPrecond: 0
                  Pf_guess: []
             MixingHistory: 7
           MixingParameter: 0.300000000000000
     MixingParameterSimple: 0.300000000000000
            PulayFrequency: 1
          PulayRestartFreq: 0
                    TWtime: 1.000000000000000e+09
                 RelaxFlag: 0
                 RelaxMeth: 'LBFGS'
            max_dilatation: 1.200000000000000
            TOL_RELAX_CELL: 0.010000000000000
                    MDFlag: 0
               RestartFlag: 0
                    MDMeth: 'NVE'
                     MD_dt: 1
                  MD_Nstep: 0
                     ion_T: -1
                thermos_TF: -1
              ion_elec_eqT: 1
              ion_vel_dstr: 2
                NLCG_sigma: 0.500000000000000
                     qmass: 1
                 L_history: 20
               L_finit_stp: 0.005000000000000
                  L_maxmov: 0.200000000000000
               L_autoscale: 1
                 L_lineopt: 1
                   L_icurv: 1
                   FIRE_dt: 1
                 FIRE_mass: 1
               FIRE_maxmov: 0.200000000000000
               Calc_stress: 1
                 Calc_pres: 0
            PrintForceFlag: 1
          PrintAtomPosFlag: 1
          PrintAtomVelFlag: 1
         PrintElecDensFlag: 0
            PrintEigenFlag: 0
                PrintMDout: 1
             PrintRelaxout: 1
              Printrestart: 1
           Printrestart_fq: 1
                       xin: 0
                       yin: 0
                       zin: 0
                     twist: 0
                  filename: 'Si2_kpt_paral'
            inputfile_path: '.'
                        XC: 'GGA_PBE'
                       Atm: [1x1 struct]
                     n_atm: 2
                     Atoms: [2x3 double]
           mvAtmConstraint: [2x3 double]
                    IsFrac: 1
                    IsSpin: 0
            is_default_psd: 0
                  temp_tol: 1.000000000000000e-12
                 xc_rhotol: 1.000000000000000e-14
                Atoms_init: [2x3 double]
                  lat_uvec: [3x3 double]
                        dx: 0.300000000000000
                        dy: 0.300000000000000
                        dz: 0.300000000000000
                        dV: 0.019091883092037
                        w2: [1x7 double]
                        w1: [1x7 double]
                         W: [15625x1 double]
                    occfac: 2
                Relax_iter: 1
              elecgs_Count: 1
               StressCount: 1
                    amu2au: 1.822888485332371e+03
                    fs2atu: 41.341373336493000
                 isOutl_11: [325x1 logical]
                 isOutr_11: [325x1 logical]
                      I_11: [325x1 double]
                     II_11: [325x1 double]
                      V_11: [325x1 double]
                 isOutl_22: [325x1 logical]
                 isOutr_22: [325x1 logical]
                      I_22: [325x1 double]
                     II_22: [325x1 double]
                      V_22: [325x1 double]
                 isOutl_33: [325x1 logical]
                 isOutr_33: [325x1 logical]
                      I_33: [325x1 double]
                     II_33: [325x1 double]
                      V_33: [325x1 double]
                  isOutl_1: [300x1 logical]
                  isOutr_1: [300x1 logical]
                       I_1: [300x1 double]
                      II_1: [300x1 double]
                       V_1: [300x1 double]
                  isOutl_2: [300x1 logical]
                  isOutr_2: [300x1 logical]
                       I_2: [300x1 double]
                      II_2: [300x1 double]
                       V_2: [300x1 double]
                  isOutl_3: [300x1 logical]
                  isOutr_3: [300x1 logical]
                       I_3: [300x1 double]
                      II_3: [300x1 double]
                       V_3: [300x1 double]
                 G_JOutl_1: [187500x1 logical]
                 G_JOutr_1: [187500x1 logical]
                     G_J_1: [187500x1 double]
                     G_I_1: [187500x1 double]
                     G_V_1: [187500x1 double]
                 G_JOutl_2: [187500x1 logical]
                 G_JOutr_2: [187500x1 logical]
                     G_J_2: [187500x1 double]
                     G_I_2: [187500x1 double]
                     G_V_2: [187500x1 double]
                 G_JOutl_3: [187500x1 logical]
                 G_JOutr_3: [187500x1 logical]
                     G_J_3: [187500x1 double]
                     G_I_3: [187500x1 double]
                     G_V_3: [187500x1 double]
                   Lap_std: [15625x15625 double]
                    grad_1: [15625x15625 double]
                    grad_2: [15625x15625 double]
                    grad_3: [15625x15625 double]
                LapPreconL: [15625x15625 double]
                LapPreconU: [15625x15625 double]
              memory_usage: 85750008
               staticfname: 'Si2_kpt_paral.static'
                 suffixNum: 0
                  outfname: 'Si2_kpt_paral.out'
                  parallel: 0
                         b: [15625x1 double]
                     b_ref: [15625x1 double]
                    rho_at: [15625x1 double]
                     Eself: 25.549411283987332
                 Eself_ref: 57.606939979716479
                       V_c: [15625x1 double]
                 PosCharge: 8.000000003757595
                 NegCharge: -8.000000003757595
                    E_corr: -3.711214099411109e-05
                       rho: [15625x1 double]
                      Atom: [1x2 struct]
                       phi: [15625x1 double]
                      e_xc: [15625x1 double]
                 dvxcdgrho: [15625x1 double]
                       Vxc: [15625x1 double]
                      Veff: [15625x1 double]
                         X: [15625x7 double]
                         F: [15625x7 double]
          mixing_hist_fkm1: [15625x1 double]
          mixing_hist_xkm1: [15625x1 double]
                       psi: [15625x10x4 double]
    upper_bound_guess_vecs: [15625x4 double]
                    EigVal: [10x4 double]
                  lambda_f: 0.217968355913433
                       occ: [10x4 double]
                    Etotal: -7.937943708116510
                     Eband: 0.027302779235666
                       Exc: -2.369643477114747
                    Exc_dc: -3.080817189163179
                  Eelec_dc: 16.873027688280551
                      Eent: -7.550124100456341e-11
                      Escc: 5.085226705875492e-07
                   S_Debug: [1x1 struct]
                     force: [2x3 double]
                 abs_force: [2x1 double]
                    Stress: [3x3 double]
                  Pressure: -7.354077637356617
             stress_dimred: [3x3 double]

