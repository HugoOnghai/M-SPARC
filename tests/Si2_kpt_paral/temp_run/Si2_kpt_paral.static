***************************************************************************
                            Atom positions                                 
***************************************************************************
Fractional coordinates of atoms:
      0.0000000000       0.0000000000       0.0000000000
      0.2500000000       0.2500000000       0.2500000000
Atomic forces (Ha/Bohr):
      0.0024428459       0.0024428203       0.0024427890
     -0.0024428459      -0.0024428203      -0.0024427890
Stress (GPa):
  7.3540823843E+00   5.4419654165E+00   5.4419705007E+00 
  5.4419654165E+00   7.3540757717E+00   5.4419852804E+00 
  5.4419705007E+00   5.4419852804E+00   7.3540747560E+00 
