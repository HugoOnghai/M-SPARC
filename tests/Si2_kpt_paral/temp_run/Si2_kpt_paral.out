***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 11-May-2021  Start time: 21:08:42                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 7.500000 7.500000 7.500000 
LATVEC:
0.500000000000000 0.500000000000000 0.000000000000000 
0.000000000000000 0.500000000000000 0.500000000000000 
0.500000000000000 0.000000000000000 0.500000000000000 
FD_GRID: 25 25 25
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 2 2 2
KPOINT_SHIFT: 5.000000e-01 5.000000e-01 5.000000e-01
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 25
NSTATES: 10
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 1
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: Si2_kpt_paral.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing                       :  0.300000 (Bohr)
Number of symmetry adapted k-points:  4
Output printed to                  :  Si2_kpt_paral.out
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  8
Atom type 1  (valence electrons)   :  Si 4
Pseudopotential                    :  Si.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  8.70 8.70 8.70
Number of atoms of type 1          :  2
Estimated total memory usage       :  81.78 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.9653531669E+00        2.660E-02        50.881
2            -3.9685232046E+00        1.410E-02        10.414
3            -3.9689599155E+00        3.418E-03        10.449
4            -3.9689700058E+00        1.781E-03        10.357
5            -3.9689717254E+00        3.237E-04        10.275
6            -3.9689718221E+00        1.686E-04        10.114
7            -3.9689718709E+00        5.058E-05        10.810
8            -3.9689718617E+00        1.433E-05        10.053
9            -3.9689718582E+00        3.426E-06        9.885
10           -3.9689718594E+00        1.421E-06        9.916
11           -3.9689718541E+00        2.232E-07        9.737
Total number of SCF: 11    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -3.9689718541E+00 (Ha/atom)
Total free energy                  : -7.9379437081E+00 (Ha)
Band structure energy              :  2.7302779236E-02 (Ha)
Exchange correlation energy        : -2.3696434771E+00 (Ha)
Self and correction energy         : -2.5549448396E+01 (Ha)
Entropy*kb*T                       : -7.5501241005E-11 (Ha)
Fermi level                        :  2.1796835591E-01 (Ha)
Average force                      :  4.2310855612E-03 (Ha/Bohr)
Maximum force                      :  4.2310855612E-03 (Ha/Bohr)
Time for force calculation         :  2.568 (sec)
Pressure                           : -7.3540776374E+00 (GPa)
Max stress                         :  7.3540823843E+00 (GPa)
Time for stress calculation        :  7.681 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  173.026 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
